Makefile

```diff
@@ -1,152 +1,245 @@
+	  ${IMAGE_MAINTAINER}/${PROJECT}-prod-{php,web,db}:latest \
+	  memcached:1.5-alpine
+
+##
+# Drupal specific commands
+# The following commands woudl change based on the version of drupal
+##
+init-drupal: drupal-install config-init config-import clear-cache
+
+update: docker-stop docker-rebuild config-import clear-cache # Run the 'rebuild' task then import configuration and clear Drupal's cache
+
+drupal-install: docker-running
+	docker exec -i ${PROJECT}-php drush \
+	  --root=/var/www/drupal site-install minimal -vv --yes \
+	  --account-name=admin \
+	  --account-pass=admin \
+	  --site-name="Drupal Dev Docker" \
+	  install_configure_form.enable_update_status_module=NULL \
+	  install_configure_form.enable_update_status_emails=NULL
+
+config-init: docker-running
 	@if [ -e ./config/system.site.yml ]; then \
 		echo "Config found. Processing setting uuid..."; \
 		cat ./config/system.site.yml | \
 		grep uuid | tail -c +7 | head -c 36 | \
-		docker exec -i cms-${PROJECT} sh -c "/var/www/vendor/bin/drush \
-		--root=/var/www/web config-set -y system.site uuid - ";\
+		docker exec -i ${PROJECT}-php drush config-set -y system.site uuid - ;\
 	else \
 		echo "Config is empty. Skipping uuid init..."; \
 	fi;
 
-config-import:
+config-import: docker-running
 	@if [ -e ./config/system.site.yml ]; then \
 		echo "Config found. Importing config..."; \
-		./bin/drush config-import sync --yes ;\
-		./bin/drush config-import sync --yes ;\
+		docker exec -i ${PROJECT}-php drush config-import sync --yes ;\
 	else \
 		echo "Config is empty. Skipping import..."; \
 	fi;
 
-config-export:
-	./bin/drush config-export sync --yes
+config-export: docker-running
+	docker exec -i ${PROJECT}-php drush config-export sync --yes
 
-config-validate:
-	./bin/drush config-export sync --no
+config-validate: docker-running
+	docker exec -i ${PROJECT}-php drush config-export sync --no
 
-config-refresh: config-init config-import
+config-refresh: config-running
 
-salt:
-	cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1 > salt.txt
+##
+# DEVELOPMENT TOOLS
+##
 
-clear-cache:
-	./bin/drush cr
+lint: # Check code for formatting or syntax errors
+	docker exec -i ${PROJECT}-php parallel-lint \
+	  -e php,module,inc,install,test,profile,theme \
+	  /var/www/drupal/modules/custom \
+	  /var/www/drupal/themes/custom
 
-destroy:
-	docker-compose -f ${DOCKER_COMPOSE_FILE} down -v
-	sudo rm -rf ./web/sites/default/files/*
-	sudo rm -rf ./web/core/*
-	sudo rm -rf ./web/libraries/*
-	sudo rm -rf ./web/modules/contrib/*
-	sudo rm -rf ./web/profiles/contrib/*
-	sudo rm -rf ./web/themes/contrib/*
-	sudo rm -rf ./drush/contrib/*
-	sudo rm -rf ./vendor/*
-
-rebuild: destroy init
-
-ready:
-	@echo "Waiting for files to sync between host and Docker...";
-	@bash ./docker-src/cms/ready.sh;
-
-lint:
-	./vendor/bin/parallel-lint -e php,module,inc,install,test,profile,theme ./web/modules/custom ./web/themes/custom
-
-sniff:
+sniff: # Description needed
```

Comment:
Might create a new /bin/tool type executable that effectively runs `docker exec -i ${PROJECT}-php [command]`. We could use it for all of these as shorthand